x-pack/solutions/observability/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -173,10 +174,11 @@ export const createMetricThresholdExecutor =
           [ALERT_GROUP]: groups,
           ...flattenAdditionalContext(additionalContext),
           ...getEcsGroups(groups),
```

Comment:
According to @benakansara's suggestion [here](https://github.com/elastic/kibana/pull/228754/files#r2222988625), we should use `groupings` object in `getEcsGroups(groups)` function